src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDescription.cs

```diff
@@ -187,6 +196,9 @@ public IEnumerable<ForeignKey> KeysToReferencedParents
             public IEnumerable<ForeignKey> KeysFromReferencingChildren
                 => db.foreignKeyLookup.KeysByReferencedParentTable[ObjectId].Select(fk => ForeignKey.Create(db, fk));
 
+            public IEnumerable<CheckConstraint> CheckConstraints
+                => db.AllCheckConstraints.Where(c => c.Table.ObjectId == this.ObjectId);

```

Comment:
Dit is niet OK, `O(n)`.

Reply:
ik heb geen idee wat je bedoeld

Reply:
Where schaalt in tijdsduur linear relatief aan de hoeveelheid constraints.